package.json

```
@@ -13,6 +13,7 @@
       "Dataset": "src/nodes/NGSI/dataset/dataset.js",
       "Updater": "src/nodes/NGSI/updater/updater.js",
       "Subscriber": "src/nodes/NGSI/subscriber/subscriber.js",
+      "Transformer": "src/nodes/NGSI/transformer/transformer.js",
```

`NGSIv2ToLD` could be a better name

---

True. "Converter" or "Translater"
```suggestion
      "Converter": "src/nodes/NGSI/converter/converter.js",
```
```suggestion
      "Translater": "src/nodes/NGSI/translater/translater.js",
```

---

what about `NGSI v2ToLD` very compact